### PR TITLE
Fix of missing initializer at sprockets.

### PR DESCRIPTION
**Pull request by vimutter**
They are using Railtie, so there are no special initializer. I've got 'immutable env' error. Which was fixed by that.

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Any document that support this change?
I am not an expert for rails initializer and I think most people are not as well.
It will be better for everyone to know what options should be used and why.

File: lib/i18n/js/engine.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 module I18n
   module JS
     class Engine < ::Rails::Engine
-      initializer "i18n-js.register_preprocessor", :after => "sprockets.environment" do
+      initializer "i18n-js.register_preprocessor", after: :engines_blank_point, before: :finisher_hook do
```
Comment (vimutter) [inline lib/i18n/js/engine.rb]: Ok, little bit more details:

1. Here is where sprockets are attached with Rails. There is no `'sprockets.environment'` mentioned.
https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb

2. Finisher hook is the place which should be used as border. 
http://guides.rubyonrails.org/configuring.html#initializers

https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb#L160

**Comment by vimutter** [comment]
@PikachuEXE Added line comment with more details.

**Comment by PikachuEXE** [comment]
I was expecting those to be put in comment, but I can add them afterwards

Also found a solid reference to "sprockets.environment"
https://github.com/rails/sprockets-rails/issues/227

Can you add a change log entry (in `bug fixes`)?

**Comment by vimutter** [comment]
@PikachuEXE Done, please check.

**Comment by PikachuEXE** [comment]
Thanks. Merging.

**Pull request merged**